src/pages/blog/MillionsInMissedFines.vue

```diff
@@ -0,0 +1,179 @@
+      <g-link
+        to="/blog"
+        class="back-link grey-link"
+      >
+        Back to Blog
+      </g-link>
+
+      <h1
+        id="main-content"
+        tabindex="-1"
+      >
+        The City Of Chicago Didn't Enforce The Building Benchmarking Ordinance, Failing to
+        Collect $30 Million In Potential Fines
+      </h1>
+      <p class="constrained bold">
+        Not Enforcing The Benchmarking Ordinance's Fines From 2018 - 2022 Reduced Accountability
+      </p>
+
+      <p class="constrained">
+        Electrify Chicago has analyzed the last five years of city benchmark data, and found
+        3,325 instances of building owners not submitting data. Based on the
+        <a
+          :href="NotifLetterUrl"
+          target="_blank"
+          rel="noopener"
+        >
+          latest (2024) official notification letter <NewTabIcon />
+        </a> fines for non-compliance could total up to $9,200 per year.
+      </p>
+
+      <img
+        src="/blog/millions-in-missed-fines/max-fine-quote.webp"
+        class="blog-img"
+        alt="A selection of text with a title 'Failure to comply is more expensive than you
+          think', with details of fines and a highlight saying 'These fines could total $9,200 /
+          year'"
+      >
+
+      <p class="caption">
+        A screenshot of the energy benchmarking notification letter's section on fines, as of
+        Feb. 2024 (<a
+          :href="NotifLetterUrl"
+          target="_blank"
+          rel="noopener"
+        >source <NewTabIcon /></a>)
+      </p>
+
+      <p>
+        With a maximum of $9,200 of fines multiplied by 3,325 instances of non-compliance
+        (<a
+          href="NonReportingBuildingsDataUrl"
+          target="_blank"
+          rel="noopener"
+        >source - City of Chicago data portal <NewTabIcon /></a>), that
+        means a whopping <strong>$30,590,000 dollars of fines could have been collected</strong>
+        by the city's commissioner of business affairs and consumer protection (who is given this
+        responsibility in the
+        <a href="https://www.chicago.gov/content/dam/city/progs/env/EnergyBenchmark/BenchmarkingOrdinance11SEP2013.pdf">
+          original ordinance
+        </a>).
+      </p>
+
+      <img
+        src="/blog/millions-in-missed-fines/non-reporting-buildings-query.webp"
+        class="blog-img"
+        alt="A screenshot of the city data portal showing a view of buildings filtered by the
+          years 2018, 2019, 2020, 2021, and 2022, and a reporting status of 'Not Reporting',
+          with the total results returned of 3,325 underlined in red"
+      >
+
+      <img
+        src="/blog/millions-in-missed-fines/zoomed-non-reporting-count.webp"
+        class="blog-img"
+        width="400"
+        alt="A screenshot of text that reads 'Showing rows 1-100 of 3325'"
+      >
+
+      <p class="caption">
+        The full data portal screenshot, and a zoomed in view of the query result count (
```

Comment:
this line seems to be not rendering properly 

![Screenshot 2024-02-28 at 4 30 14 PM](https://github.com/vkoves/electrify-chicago/assets/919583/373e962e-594f-4287-b7a4-ed9dc6cd8997)